packages/compass-utils/src/electron.ts

```diff
@@ -21,12 +17,10 @@ function getElectronApp() {
   return app;
 }
 
-export function getStoragePaths(): StoragePaths | undefined {
-  const app = getElectronApp();
-  if (!app) return undefined;
-
-  const appName = app.getName();
-  const basepath = app.getPath('userData');
+export function getAppName(): string | undefined {
+  return getElectronApp()?.getName();
+}
 
-  return { appName, basepath };
+export function getAppPath(): string | undefined {
```

Comment:
Actual app path is not the same as user data path, I think it might create some confusion about what is being returned here